misc/wiki/Parameters.md

```diff
@@ -56,7 +57,7 @@ Write out the &chi;<sub>0</sub> function of the BSE lattice solver.
     {
         "output": {
             "directory": "./T=0.5",
-            "output-format": "HDF5",
```

Comment:
output-format is just about to have another choice ADIOS2 added, this can't be eliminated.  I don't recall any discussion of removing the JSON capability either.

Reply:
I see. We did not discuss JSON, but we wanted to add writing the computation state  after every DCA loop. 
It is not possible to overwrite a field on the (rather crude) JSON writer. As none uses it and it is anyway not suited to write an output as large as a DCA run, I thought the easiest thing was to remove it. Avoiding the logging of the state if the writer is json is another easy solution, maybe slightly hackish. Less easy would be using a proper writer from a library.
But the important thing is that it is compatible with the ADIOS2 integration.